File: python/26_RemoveDuplicatesFromSortedArray.py
class Solution:
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        
        if len(nums) == 0:
            return 0
        
        j = 0
        
        for i in range(0, len(nums) - 1):
            if nums[i] != nums[i+1]:
                nums[j] = nums[i]
                j += 1
                
        nums[j] = nums[len(nums) - 1]
        j += 1
        return j
            

        

File: C#/26_RemoveDuplicatesFromSortedArray.cs
public class Solution {
     public int RemoveDuplicates(int[] arr){
        if (arr.Length == 0)
            return 0;
        
        int j=0;
 
        for (int i = 0; i < arr.Length - 1; i++)
            if (arr[i] != arr[i + 1]) arr[j++] = arr[i];
            
        arr[j++] = arr[arr.Length-1] ;

        
        return j;
     
        }
}

